Client/Frontend/Home/TopSites/DataManagement/ContileProvider.swift

```diff
@@ -68,9 +68,8 @@ class ContileProvider: ContileProviderInterface, Loggable, URLCaching, FeatureFl
             }
 
             guard let response = validatedHTTPResponse(response, statusCode: 200..<300),
-                  let data = data, !data.isEmpty
-            else {
-                self.browserLog.debug("Response isn't proper: \(response.debugDescription), with data \(String(describing: data))")
+                  let data = data else {
```

Comment:
```suggestion
                  let data = data else {
```
Nit: `else` should be on a new line to help differentiate.